CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
   Funds with a hold on them cannot be transferred until the hold is removed.
   Management of holds is internal, but there are queries for looking up holds on accounts.
   Holds are also reflected in the `x/bank` module's `SpendableBalances` query.
+* Add new MaxSupply param to marker module, and deprecate MaxTotalSupply. [#1292](https://github.com/provenance-io/provenance/issues/1292).
```

Comment:
Is there a 3rd item that needs to be listed here or is the comma left over?

Reply:
Whoops typo. Thanks for finding it.